### ISSUE TITLE
Error when try to import the package

### ISSUE THREAD

**Issue by Elvincth** [hwchase17/langchainjs#382]
I try to import `import { ChatOpenAI } from "langchain/chat_models";`

But got an error:
`Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './chat_models' is not defined by "exports" `

**Comment by nfcampos** [comment]
Hi, that error suggests you're missing `"type": "module"` in your package.json, you can see the full install instructions here https://hwchase17.github.io/langchainjs/docs/install

**Comment by GJmelody** [comment]
I get error  `Unknown file extension ".ts"` after adding "type": "module"

**Comment by asibilia** [comment]
Same issue here. Even with `"type": "module"` in my `package.json` file, I get the error:

> Package subpath './<submodule_name>' is not defined by "exports" 

For whichever module is imported first. In my case, it's `agents`

**Comment by Marviel** [comment]
[ERROR] 15:33:01 Error: Must use import to load ES Module: /Users/lukebechtel/localdev/lib/apps/scripts/src/agents/simple-notion-agent.ts

**Comment by Marviel** [comment]
I ended up copying this package.json from the examples directory, and things started working again:
https://github.com/hwchase17/langchainjs/blob/main/examples/package.json

Not a viable long-term solution though.

Using the `tsx` packages instead of `ts-node` also helped me track down another issue I was facing.

**Comment by nfcampos** [comment]
@Marviel that error means that your tsconfig was setup to output CJS, see the instructions here https://js.langchain.com/docs/getting-started/install#typescript

@atulmy can you share more about your setup? 

@asibilia I'd suggest opening an issue on the `next-with-linaria` repository. As for the peer dependencies, sadly it seems some tools in the JS ecosystem are implemented incorrectly and treat peer dependencies / dynamic imports as if they were required. There are two solutions for working with these tools (it appears nextjs is one of them?) 
1. You can install the peer dependencies in your project
2. or, You can mark the peer dependencies as `external` in the tool config. For nextjs this would be in https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config. eg. 
```js
module.exports = {
  webpack: (
    config,
    { buildId, dev, isServer, defaultLoaders, nextRuntime, webpack }
  ) => {
    config.externals = {
      'typeorm': 'typeorm',
      // add others here
    }
    return config
  },
}

**Comment by atulmy** [comment]
@nfcampos 

Executing:
`ts-node ./src/index.ts`

tsconfig:
```
"target": "esnext",
"module": "commonjs",
```

**Comment by nfcampos** [comment]
@atulmy your issue might be fixed by changing to `"module": "nodenext",`

**Comment by nfcampos** [comment]
Hi, we've released a new version 0.0.49 which adds a CJS build and fixes this issue.

**Comment by whitekanegon** [comment]
Changing the import statement to the following worked:

import { ChatOpenAI } from 'langchain/chat_models/openai';
import { AIMessage, HumanMessage } from 'langchain/schema';
↓↓↓
import { ChatOpenAI } from '@langchain/openai';
import { HumanMessage, AIMessage } from '@langchain/core/messages';
